sdk/keyvault/azure-security-keyvault-keys/test/ut/key_rotation_policy_test_live.cpp

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "gtest/gtest.h"
+
+#include "../../src/private/key_serializers.hpp"
```

Comment:
It is some way better to include the folder that contains the private headers with CMake project than handling the path manually.
See how we include private headers in azure core tests here: https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core/test/ut/CMakeLists.txt#L100

Then you can include the headers as `#include "private/key_serializer.hpp" and there will be only one place (CMake) to be updated if the path changes

Reply:
done